addons/assistance.py

```diff
@@ -243,24 +243,23 @@ def check_console(self, message, channel, consoles):
         embed.description = "1. FBI\n2. Remote Install\n3. Recieve URLs over the network"
         embed.add_field(name="Check your 3DSs IP (Homebrew)", value="1. Open Homebrew Launcher\n2. Press Y")
         await self.bot.say("", embed=embed)
-
+        
     @commands.command(pass_context=True)
     @commands.cooldown(rate=1, per=30.0, type=commands.BucketType.channel)
     async def stock(self,ctx,console=""):
         """Advisory for various Nintendo systems on stock firmware"""
         if self.check_console(console, ctx.message.channel.name, '3ds'):
             embed = discord.Embed(title="Running stock (unmodified) 11.4+ firmware?", color=discord.Color.dark_orange())
             embed.description = cleandoc("""
-                You have 4 possible options for installing CFW:
-                - [NTRBoot](https://3ds.hacks.guide/ntrboot) which requires a compatible NDS flashcart and maybe an\
-additional DS(i) or hacked 3DS console depending on the flashcart
-                - [Frogminer](https://3ds.filthycasuals.tech/frogminer.html) which requires a homebrew entrypoint like \
-[steelminer](http://steelminer.jisagi.net/) (free method) or freakyhax.
-                - [Seedminer](https://3ds.hacks.guide/installing-boot9strap-\(seedminer\)) which requires a compatible \
-DSiWare game
-                - [Hardmod](https://3ds.hacks.guide/installing-boot9strap-\(hardmod\)) which requires "soldering \
-**Not for beginners!**
-                **Downgrading is impossible on 11.4+!**
+                You have many possible options for installing CFW:
```

Comment:
![i](https://files.frozenchen.me/8ZX3W.png)
https://www.python.org/dev/peps/pep-0008/#maximum-line-length
The change to multilines was for this very reason. At most 120 chars per line

Reply:
Ya got me